cleaning_data.py

```diff
@@ -0,0 +1,199 @@
+"""change documents to UTF-8"""

```

Comment:
in python 2, we were using heading about encoding
in python 3, file can start by comment explaining purpose of file
or by imports